packages/react-app/src/components/field-input/date-field-input.tsx

```diff
@@ -1,28 +1,42 @@
 import { useTheme } from '@1hive/1hive-ui';
 import { connect } from 'formik';
 import { noop } from 'lodash-es';
-import { CSSProperties, ReactNode } from 'react';
+import { CSSProperties, FocusEventHandler, ReactNode } from 'react';
 import { GUpx, isDarkTheme } from 'src/utils/style.util';
-import styled from 'styled-components';
+import styled, { css as _css } from 'styled-components';
```

Comment:
Using `css` shadown the other variable with the same name. One good practice is put all styled in another file called `.styled.ts`

Reply:
Yeah might be a huge refactor for that one, maybe add a card in the Icebox column

Reply:
#220 https://app.zenhub.com/workspaces/quests-6092dda4c272a5000e858266/issues/1hive/quests/220